docs/CreateDialog.md

```diff
@@ -77,7 +77,7 @@ In the related `<Resource>`, you don't need to declare a `create` component as t
 | `children`         | Required | `ReactNode` |         | The content of the dialog                                       |
 | `fullWidth`        | Optional | `boolean`   | `false` | If `true`, the dialog stretches to the full width of the screen |
 | `maxWidth`         | Optional | `string`    | `sm`    | The max width of the dialog                                     |
-| `mutation Options` | Optional | `object`    |         | The options to pass to the `useMutation` hook                   |
+| `mutationOptions`  | Optional | `object`    |         | The options to pass to the `useMutation` hook                   |
```

Comment:
I believe the extra space was added on purpose to avoid the first column taking too much horizontal space because of the unbreakable word. I'm not against removing it but @fzaninotto might be.